src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1694,7 +1725,7 @@ bool AdaptDispatch::_EraseAll()
 // - fEnabled - set to true to allow DECCOLM to be used, false to disallow.
 // Return Value:
 // - True if handled successfully. False otherwise.
-bool AdaptDispatch::EnableDECCOLMSupport(const bool fEnabled)
+bool AdaptDispatch::EnableDECCOLMSupport(const bool fEnabled) noexcept
```

Comment:
This `f` didn't get ~deplatformed~ unprefixed

Reply:
thx